packages/core/lib/queues/AbstractPublisherManager.ts

```diff
@@ -115,6 +115,7 @@ export abstract class AbstractPublisherManager<
     return {}
   }
 
+  // biome-ignore  lint/correctness/noUnusedPrivateClassMembers: this is used in constructor
```

Comment:
biome seems to be too eager to flag things that are actually used, hopefully they will fix it later, but skipping for now